db.json

```diff
@@ -0,0 +1,25 @@
+{
+  "events": [
+    {
+      "id": "1722321102992-4475",
+      "eventTitle": "pirmas",
+      "startDateTime": "Tue Jul 30 2024 09:00:00 GMT+0300 (Eastern European Summer Time)",
+      "endDateTime": "Tue Jul 30 2024 10:00:00 GMT+0300 (Eastern European Summer Time)",
+      "description": "qqqqqqq"
```

Comment:
if committed to github text should be not necessarily very meaningful, but at least they should be texts : )) like Event Title 1, Test 1, Lorem ipsum.., etc., same with description.

Also, should description be always required?

I have also noticed the way date is stored. It would be better to store it in more standardised way. Check [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601)